src/client/pages/ToeristischeVerhuur/ToeristischeVerhuur.tsx

```diff
@@ -50,11 +49,17 @@ export default function ToeristischeVerhuur() {
     const vergunningen = [];
 
     for (const vergunning of content.vergunningen) {
+      const showExpired =
+        !vergunning.isActual &&
+        vergunning.decision !== 'Ingetrokken' &&
+        vergunning.title === 'Vergunning vakantieverhuur';
       const displayVergunning = {
         ...vergunning,
         status:
           vergunning.status === 'Afgehandeld'
-            ? vergunning.decision ?? vergunning.status
+            ? showExpired
```

Comment:
We should try to avoid these nested ternary nullish coalsecing constructs. I think it's better to define the label on in the top of the loop statement.